src/main/java/org/gridsuite/study/server/NetworkModificationService.java

```diff
@@ -293,4 +293,17 @@ private String buildPathFrom(UUID networkUuid) {
             .retrieve()
             .bodyToMono(Void.class);
     }
+
+    public Mono<Void> deleteModification(UUID groupUuid, UUID modificationUuid) {
+        Objects.requireNonNull(groupUuid);
+        var path = UriComponentsBuilder.fromPath("groups" + DELIMITER + "{groupUuid}"
+                + DELIMITER + "modifications" + DELIMITER + "{modificationUuid}")
+            .buildAndExpand(groupUuid, modificationUuid)
+            .toUriString();
+        return webClient.delete()
+            .uri(getNetworkModificationServerURI(false) + path)
+            .retrieve()
+            .onStatus(httpStatus -> httpStatus == HttpStatus.NOT_FOUND, r -> Mono.empty()) // Ignore because modification group does not exist if no modifications
```

Comment:
There is no error handling like for DELETE_EQUIPMENT_FAILED for example ?

Reply:
we don't apply modification we just delete it, then the build is invalidated

Reply:
I dmn't understand. Should we add StudyException::Type.MODIFICATION_NOT_FOUND and DELETE_MODIFICATION_FAILED and manage error handling here using handleChangeError or not ?